coral-service/src/main/java/com/linkedin/coral/coralservice/entity/VisualizationResponseBody.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.coralservice.entity;
+
+import java.util.UUID;
+
+
+public class VisualizationResponseBody {
+
+  private UUID sqlNodeImageID;
+  private UUID relNodeImageID;
+  private UUID postRewriteSqlNodeImageID;
+  private UUID postRewriteRelNodeImageID;
+  public VisualizationResponseBody() {
+  }
+
+  public UUID getSqlNodeImageID() {
+    return sqlNodeImageID;
+  }
+
+  public UUID getRelNodeImageID() {
+    return relNodeImageID;
+  }
+
+  public UUID getPostRewriteSqlNodeImageID() {
+    return postRewriteSqlNodeImageID;
+  }
```

Comment:
Are you going to use this in follow up PRs? I am not clear if the `images` API works by looking up an ID, how do we know what type of image is it (rel/sql + pre-post rewrite)?

Reply:
An example of a payload returned by /generategraphs looks like this:

`{"sqlNodeImageID":"804290e5-0523-44e5-9c03-952a0ec4c3d3","relNodeImageID":"52e985b7-6b70-47a0-aa18-9dcc0afbbbee","postRewriteSqlNodeImageID":"418d17a6-e337-4433-a365-0e533fec7ebd","postRewriteRelNodeImageID":"e594dcd6-3b9b-4012-82e7-a2ac419f55aa"}`

Once the frontend has these values its a simple fetch w the ids as params to get the images, and knows the type of image based on the field name.

This can definitely be used in future PRs

Reply:
If no rewrite is requested postRewriteRelNodeImageID and postRewriteSqlNodeImageID have `null` values.